flight/targets/pipxtreme/board-info/board-info.mk

```diff
@@ -1,5 +1,5 @@
 BOARD_TYPE          := 0x03
-BOARD_REVISION      := 0x01
+BOARD_REVISION      := 0x02
```

Comment:
I'm confused, why is this here? are you using the pipx board file for the taulink?